src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -647,16 +647,16 @@ export class CPlusPlusRenderer extends ConvenienceRenderer {
 
     protected emitEnum(e: EnumType, enumName: Name): void {
         const caseNames: Sourcelike[] = [];
-        const enumValues = enumCaseNames(e, this.targetLanguage.name );
+        const enumValues = enumCaseValues(e, this.targetLanguage.name );
 
         this.forEachEnumCase(e, "none", (name, jsonName) => {
             if (caseNames.length > 0) caseNames.push(", ");
             caseNames.push(name);
 
             if (enumValues !== undefined) {
-                const [ enumvalue, isFixed ] = getAccessorName(enumValues, jsonName);
-                if (enumvalue !== undefined && isFixed !== undefined) {
-                    caseNames.push(" = " + enumvalue);
+                const [ enumValue, ] = getAccessorName(enumValues, jsonName);
```

Comment:
Oh, I wasn't aware you could do this - I would always do `const bla = something()[0];`.  Nice! :-)